Add torch.tensor to named_data_store

lucylq opened the pull request:
Summary: Allow adding torch.Tensor to named_data_store, and infer the TensorLayout.

Differential Revision: D85992938

larryliu0820 reviewed (approved):
Review automatically exported from Phabricator review in Meta.

lucylq committed:
Add torch.tensor to named_data_store (#15504)

Summary:

Allow adding torch.Tensor to named_data_store, and infer the TensorLayout.

Reviewed By: larryliu0820

Differential Revision: D85992938
